lib/github_s.mli

```diff
@@ -737,6 +737,15 @@ module type Github = sig
         {{:https://developer.github.com/v3/git/refs/}git references}
         with prefix [?ty] for repo [user]/[repo]. *)
 
+    val get_ref :
+      ?token:Token.t ->
+      user:string -> repo:string ->
+      name:string ->
+      unit -> Github_t.git_ref Response.t Monad.t
```

Comment:
I'd really like to get rid of those 404 errors everywhere, eventually. I guess this is OK for now but I'd appreciate an issue that starts to track all of the horrible 404 exceptions for 3.0.